gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,91 +1,151 @@
+
+An optional external REDIS storage can be configured for features such as distributed rate limiting and distributed caching. Gateway replicas use the REDIS storage as a scratchpad to coordinate rate-limiting across multiple replicas, so customer SLAs are adhered to.
+
+== Connected Mode vs. Local Mode
+
+Flex Gateway can be deployed in two ways:
+
+* Connected Mode
+* Local Mode
+
+=== Connected Mode
+In Connected Mode, the gateway is fully connected to the MuleSoft control plane. This allows for centralized management, observability, and security. Anypoint API Manager enables full API lifecycle management and policy configuration. Anypoint Runtime Manager enables you to deploy and configure your gateway.
+
+Choose Connected Mode for a UI-based experience to deploy policies, and for managing and monitoring the gateway.
+
+=== Local Mode
+
+You can also configure and manage a standalone gateway that is mostly disconnected from the control plane. Choose Local Mode for this experience.
+
+[NOTE]
+====
+Flex Gateway deployed in Local Mode only connects to the control plane for registration and logging usage metrics.
+====
+
+In Local Mode, you manage all configuration and policy applications with locally stored declarative configuration files.
 
-== Anypoint Service Mesh
+Use Local Mode to build CI/CD pipelines for application deployments.
 
-Anypoint Service Mesh enables you to extend your microservices network by including your non-MuleSoft applications into the Anypoint Platform sphere. You can then manage and secure your application network, including your non-Mule applications, seamlessly from a single pane.
+=== Summary of Differences
 
-Your non-MuleSoft microservices might be written using different languages and platforms, or might be deployed in different environments. To alleviate issues arising from differences in these microservices, Anypoint Service Mesh ensures the following application management advantages:
+The following table summarizes the differences between Connected Mode and Local Mode.
 
-* Provides resilient service-to-service communication using zero-trust policies to handle traffic control, fault tolerance and load balancing
-* Extends MuleSoft API Management capabilities to any non-Mule service, with policies, security and analytics all applied directly from Anypoint Platform
-* Amplifies reuse of services with discovery of non-Mule services within Exchange, available for use in future projects
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| | Connected Mode | Local Mode
 
-For more information, see xref:service-mesh::getting-started-service-mesh.adoc[Get Started with Service Mesh].
+| *Use Case*
+| Choose Connected Mode for centralized management, observability, and security. Flex Gateway in Connected mode connects to the control plane.
```

Comment:
Connected Mode or connected mode

Reply:
I think, since you have the modes in the column heads, you don't need to say "Choose xxxx". 

The description seems repetitive with the previous descriptions. 

Maybe split the use-case descriptions into smaller chunks:
Use Case
Management

Reply:
Fixed